docs/evidence/characters/hydro/candace.md

```diff
@@ -10,16 +10,66 @@ search: false
+  * [Imgur](https://imgur.com/a/CXxGhBq)
+  * [Imgur](https://imgur.com/a/t2Wl7w0)
+  * [Imgur](https://imgur.com/a/gW7sxZH)
+* Candace's counter has no startup, it is active as soon as the UI greys out. However, there is a variable delay between key press and activation dependent on input delay.- Aftermath#7658
+  * [Imgur](https://imgur.com/Xpn2eVs)
+* Tap E creates 2 particles each time it hits an enemy. - Puffin\#9920
+  * 16 trials, no case where 2 particles are not produced - [Video](https://youtu.be/8HMzZfMOZOc)
+* Hold E creates 3 particles each time it hits an enemy. - Puffin\#9920
+  * 5 minutes' worth of trials, should be enough - [Video](https://youtu.be/VUZTlHKVq64)
+
 ## Burst Mechanics
 
 ## Ascension Mechanics
 
+### A1: Aegis of Crossed Arrows
+* Her A1 will only activate while E is held at the time of getting hit. - Aftermath\#7658
+  * [Imgur](https://imgur.com/viN0iTM)
+  * [Imgur](https://imgur.com/IILpMea)
+
 ## Constellation Mechanics
 
 ## Synergies/Interactions
+
+* Candace's infusion overrides c6 Bennett. - [Video](https://www.youtube.com/watch?v=xXE9ZbcmfIo) - Sew\#2409
+* Candace can't overwrite Chongyuns infusion. - [Video](https://www.youtube.com/watch?v=QjD49OckiVc) - Sew\#2409
+* Candace + Chongyun infusion = frozen weapon (that does Cryo dmg) - [Video](https://youtu.be/N-ThOhYi6X8) - Mystathi#9705
+  * Significance: Chongyun doesn’t quite override Candace. What may happen is that the weapon infusions trigger freeze on the weapon, which depletes both elements and leaves a frozen aura. This frozen aura is what deals Cryo dmg. This is more information on weapon infusions and can lead into questions of dumb shit like can a weapon do Cryo and hydro dmg if there is enough hydro on it (like how swirl can swirl Cryo and hydro on a frozen enemy).
+* Interaction between Candace and Keqing infusion results in weapon being Electrocharged but deals=ing hydro dmg. - Nass008\#8577
+  * After keqing cast her skill, her weapon gains purple aura as usual but she still has visual effect of Candace burst.
+When using elemental sight, her weapon looks like electro infused. - [Video](https://imgur.com/W0LdtNj)
+  * Even though her weapon has appearance of electro infusion, Keqing normal attacks under Candace burst deals hydro dmg. - [Video](https://imgur.com/WTnZFQY)
+  * When applying infusions in order of “Candace burst -> Keqing skill -> Chongyun skill”. Super conduct is triggered. Indicating electro gauge is physically present (not just as a visual effect) even though it does not seem to affect damage type of attacks. - [Video](https://imgur.com/hjHzlpY)
+* Candace n4 string is able to get 12 procs of Beidou's Q pretty easily. Her Q and E timings allow for multiple variations. These combos do require a small amount of timing after the N4, since mashing LMB or dashing into an N1 will make you hit slightly too fast. - [Video](https://youtu.be/sGAyBp8IA5k)
```

Comment:
```suggestion
* Candace N4 string is able to get 12 procs of Beidou's Q pretty easily. Her Q and E timings allow for multiple variations. These combos do require a small amount of timing after the N4, since mashing LMB or dashing into an N1 will make you hit slightly too fast. - [Video](https://youtu.be/sGAyBp8IA5k)
```